torchaudio/models/wavernn.py

```diff
@@ -1,18 +1,43 @@
-from typing import List, Tuple
+from typing import List, Tuple, Dict, Any
 
 import torch
 from torch import Tensor
 from torch import nn
+try:
+    from torch.hub import load_state_dict_from_url
+except ImportError:
+    from torch.utils.model_zoo import load_url as load_state_dict_from_url
```

Comment:
Is it necessary to try `torch.hub` first?
`from torch.utils.model_zoo import load_url` seems to suffice.

Reply:
Looking at the document, the code has been moved to `torch.hub.load_state_dict_from_url`.
`torchaudio` only supports the matching PyTorch that is released at the same time, so only `from torch.hub import load_state_dict_from_url` should work.

Reply:
Thank you for pointing it out, I didn't realize that.
It's addressed [here](https://github.com/pytorch/audio/pull/1612/files#diff-7f4ad259da182d3840fe4c75dd0c0b7b9fe30f31503df0eff87b1e5c9378cac4R6)